.pre-commit-config.yaml

```diff
@@ -28,6 +28,7 @@ repos:
         - types-requests==2.30.0
         - types-tabulate==0.9.0.20240106
         - types-toml==0.10.8.20240310
+      exclude: ^tasks/unit_tests/.*$
```

Comment:
```suggestion
      exclude: ^tasks/unit-tests/.*$
```

Reply:
It got renamed last Friday to `unit_tests` 🙂